ui/app/AppLayouts/Chat/panels/EmptyChatPanel.qml

```diff
@@ -20,7 +20,7 @@ Item {
         id: walkieTalkieImage
         anchors.horizontalCenter: parent.horizontalCenter
         anchors.verticalCenter: parent.verticalCenter
-        source: Style.png("onboarding/chat@2x")
+        source: Style.png("chat/chat@2x")
```

Comment:
@alexandraB99 I am thinking about using 2x and 3x. According to docs, on Mac, you need to use only "chat/chat" and the system will automatically choose 2x or 3x. Here you explicitly choose 2x. Do you have any thoughts ?

Reply:
@endulab I don't. this was there from before, I just changed the path to it :) Fingerprint is also used like this `source: Style.png("onboarding/fingerprint@2x")` in TouchIDAuthView, maybe @borismelnik knows something?